spring-kafka/src/test/java/org/springframework/kafka/core/DefaultKafkaConsumerFactoryTests.java

```diff
@@ -152,6 +155,15 @@ public void testContainerTxProducerIsNotCached() throws Exception {
 			pfTx.destroy();
 		}
 	}
+	
+	@Test
+	public void testKafkaEmbedded() {
+		assertThat(embeddedKafka.getBrokersAsString()).isEqualTo("127.0.0.1:" + 8085);
```

Comment:
I somehow don't like this test against an explicit port and an attempt for real Kafka broker on it.

Can't we somehow have a mocking test do not open this port in the reality?
I mean there is just enough to be sure that `ports` is populated properly into the `EmbeddedKafkaBroker.kafkaPorts` without starting a broker(s).